cmp/grid/columns/Column.js

```diff
@@ -222,7 +222,7 @@ export class Column {
         }
 
         if (this.tooltip) {
-            ret.tooltip = isFunction(this.tooltip) ?
+            ret.tooltipValueGetter = isFunction(this.tooltip) ?
```

Comment:
Thanks for this - I was going to suggest it, as I knew we had made then unwound this change.  But then I went to look at their docs and couldn't find the reference again to this change, and tooltips still seemed to be working without....  Were you able to find a pointer on their site to this change?  

(I believe it's the right thing to do based on those recollections - just want to be sure.)

Reply:
Didn't see it in the changelog, but noticed some deprecation warnings in the console, and it is marked as deprecated as of v20.1 in the ag-grid source code. A quick search through the ag-grid docs doesn't actually turn up any references to `tooltipValueGetter` at all ... but pretty sure this is correct